aristanetworks/ansible-avd
# Test: New cvp integration molecule test

## Change Summary

Molecule test for the new CVP integration

## Related PR(s)
#3466 

## How to test
NOTE: Can only be run on `aawg-ci` tenant. 
1. Create a service account token on cvaas
2. Load the token as environment variable on your local machine
3. `bash# export CVAAS_AAWG_CI=<your token here>`
4. `bash# cd avd/ansible_collections/arista/avd`
5. `bash# molecule test -s cv_workflow
6. You can add the `-- -vvv` option to get extended logs
7. Verify that the molecule test completes without any errors
